CounterpointGenerator/CounterpointGenerator/MelodyLine.cs

```diff
@@ -34,8 +36,21 @@ public MelodyLine()
         {
             this.AMelodyLine = new List<Note>();
         }
+
+        public MelodyLine(string cantus)
+        {
+            List<Note> InputCantus = new List<Note>();
+            List<string> CantusString = cantus.Split(' ').ToList();
```

Comment:
You may prefer the functional style where this entire function can be written as

```
this.AMelodyLine = cantus.Split(' ')
    .Select(item => new Note(item))
    .ToList();
```

Reply:
that's concise and clean!